Sources/Maker/UIViewMaker.swift

```diff
@@ -0,0 +1,235 @@
+//
+//  Copyright © 2020 Rosberry. All rights reserved.
+//
+
+import UIKit
+
+public final class UIViewMaker: Maker {
```

Comment:
Naming is incorrect, this class is not Obj-C and it doesn't belong to UIKit namespace.